server/graphql.server.ts

```diff
@@ -19,26 +21,33 @@ export interface IPolarisGraphQLServer {
 }
 
 @injectable()
-export class PolarisGraphQLServer implements IPolarisGraphQLServer{
+export class PolarisGraphQLServer implements IPolarisGraphQLServer {
     private server: ApolloServer;
     private _polarisProperties: PolarisProperties;
     private _logProperties: LoggerConfiguration;
-    @inject("InjectableLogger")polarisLogger :InjectableLogger;
+    @inject("InjectableLogger") polarisLogger: InjectableLogger;
 
-    constructor(@inject("ISchemaCreator")creator :ISchemaCreator,
-                @inject("ILogConfig") logConfig: ILogConfig,
-                @inject("IPolarisServerConfig") propertiesConfig: IPolarisServerConfig) {
+    constructor(
+        @inject("ISchemaCreator")creator: ISchemaCreator,
+        @inject("ILogConfig") logConfig: ILogConfig,
+        @inject("IPolarisServerConfig") propertiesConfig: IPolarisServerConfig,
+        @multiInject("PolarisMiddleware") middlewares: PolarisMiddleware[]
+    ) {
         let schema = creator.generateSchema();
         let executableSchemaDefinition: { typeDefs: any, resolvers: any } = {
             typeDefs: schema.def,
             resolvers: schema.resolvers
         };
         let executableSchema = makeExecutableSchema(executableSchemaDefinition);
 
+        const schemaWithMiddleware = applyMiddleware(
```

Comment:
rename to executableSchemaWithMiddlewares

Reply:
done